airbyte-integrations/connectors/destination-e2e-test/src/test-integration/java/io/airbyte/integrations/destination/e2e_test/TestingSilentDestinationAcceptanceTest.java

```diff
@@ -1,79 +1,79 @@
-/*
- * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
- */
-
-package io.airbyte.integrations.destination.e2e_test;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
-import com.fasterxml.jackson.databind.JsonNode;
-import io.airbyte.cdk.integrations.standardtest.destination.DestinationAcceptanceTest;
-import io.airbyte.commons.json.Jsons;
-import io.airbyte.integrations.destination.e2e_test.TestingDestinations.TestDestinationType;
-import io.airbyte.protocol.models.v0.AirbyteMessage;
-import io.airbyte.protocol.models.v0.AirbyteRecordMessage;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import org.junit.jupiter.api.Disabled;
-import org.junit.jupiter.api.Test;
-
-public class TestingSilentDestinationAcceptanceTest extends DestinationAcceptanceTest {
```

Comment:
I didn't check what these two classes are doing, but will revive these tests before merging this PR